tests/integration/ambient/waypoint_test.go

```diff
@@ -381,20 +381,26 @@ func TestWaypointDNS(t *testing.T) {
 				if src.Config().HasSidecar() {
 					t.Skip("TODO: sidecars don't properly handle use-waypoint")
 				}
+				address := "240.240.240.239"
```

Comment:
We can also fixup baseline_test L1742 and L1913

Reply:
I will look at those tests.